client/src/app/components/page/list/list.component.ts

```diff
@@ -34,7 +34,14 @@ export class ListComponent implements OnInit, OnDestroy {
             takeUntil(this.ngUnsubscribe)
         ).subscribe(
             tasks => {
-                this.tasks = tasks;
+                this.tasks = tasks.map(task=>{
```

Comment:
If you are going to modify (mutate) array element, please use `forEach` function. `map` should be "pure" function, so its job should be only to iterate and return new array. Here you are modify array you are mapping, which is called "side effect" and it is discouraged within `map` or any "pure" function.

Anyway you don't need this modification for `createdAt` and `updatedAt`. Just use angular date pipe for display.

Reply:
Deleted